docs/DESIGN.md

```diff
@@ -1,67 +1,81 @@
-## Pinterest DoctorKafka Design
+# Pinterest DoctorKafka Design
 
-#### High Level Design
+## Components
 
-DoctorKafka is composed of two parts: 
+The DoctorKafka system is composed of two parts: 
+  1.  **KafkaStats Agent**: Metrics collector that is deployed on every kafka broker
+  2.  **DoctorKafka Service**: Central operation automation logic that analyzes broker status and execute kafka operation commands
 
-  * Metrics collector that is deployed to each kafka broker
-  * Central doctorkafka service that analyzes broker status and execute kafka operation commands
+The metric collectors send metrics to a kafka topic that the central DoctorKafka service read from. DoctorKafka takes actions and also log its action to another topic that can be viewed through web UI.
```

Comment:
concatenate this to line 6? 

      Metrics collector that is deployed on every kafka broker.  It sends metrics to a kafka topic 
      that the central DoctorKafka service read from. DoctorKafka takes actions and also log its 
      action to another topic that can be viewed through web UI.